packages/odf/components/storage-consumers/onboarding-modal.tsx

```diff
@@ -77,12 +77,23 @@ export const ClientOnBoardingModal: ClientOnBoardingModalProps = ({
             </Button>
           </FlexItem>
           <FlexItem>
-            <Text component={TextVariants.h6}>How to use this token?</Text>
-            <Text>Short explanation</Text>
+            <Text component={TextVariants.h6}>
+              {t('How to use this token?')}
+            </Text>
+            <Text>
+              {t(
+                'To onboard the client cluster, the provider cluster requires the onboarding token.'
+              )}
+            </Text>
+            <Text>
+              {t(
+                'Copy the generated token and add it to the storage client CR from the command-line interface'
```

Comment:
since it's a `kind`...
```suggestion
                'Copy the generated token and add it to the StorageClient CR from the command-line interface'
```